### ISSUE TITLE
[FEATURE] Styling the rent calculator on home page

### ISSUE THREAD

**Issue by KajalDeore04** [gauravsingh1281/Rentalog.in--Frontend#642]
I want to style the rent calculator to match the theme

### Checklist  
- [* ] I have checked the existing issues and this feature request is not a duplicate.  
- [ *] I have **starred this repository**.  
- [* ] I am willing to help implement this feature (optional).

Please assign me this under SWOC

**Comment by gauravsingh1281** [comment]
Hey @KajalDeore04 contribution under SWOC 25 starts after 1 Jan 2025.

**Comment by KajalDeore04** [comment]
Yeah! sorry, I thought we can open an issue before that. 😢 Would you please assign me this once it starts ?

**Comment by gauravsingh1281** [comment]
@KajalDeore04 ok

**Comment by gauravsingh1281** [comment]
Hey @KajalDeore04 ok take inspiration from rentalog.in for rent-related services.

**Comment by KajalDeore04** [comment]
hey @gauravsingh1281  can you now assign me this?

**Comment by KajalDeore04** [comment]
@gauravsingh1281 please add the SWOC label with a level label

**Comment by gauravsingh1281** [comment]
@KajalDeore04 are you working on this ?

**Comment by gauravsingh1281** [comment]
@KajalDeore04 ?

**Comment by KajalDeore04** [comment]
instructions were not that clear so... No, I am not working on it.

**Comment by gauravsingh1281** [comment]
@KajalDeore04 ok I just wanted you to remove the rent calculator features from the homepage. and users can access it through the admin dashboard after login. like the original site rentalog.in
